com.valvesoftware.SteamLink.appdata.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.valvesoftware.SteamLink</id>
+  <launchable type="desktop-id">com.valvesoftware.SteamLink.desktop</launchable>
+  <metadata_license>MIT</metadata_license>
+  <project_license>LicenseRef-Proprietary</project_license>
+  <name>Steam Link</name>
+  <summary>Stream games from another computer with Steam</summary>
```

Comment:
The metainfo needs at least one screenshot

Reply:
Fixed. It would be great if the application submission info explicitly said either this, or which validator(s) the metainfo must pass (or perhaps it does and I missed it).

Reply:
Our documentation isn't great for now, but we have the appdata guidlines as part of the wiki https://github.com/flathub/flathub/wiki/AppData-Guidelines along with how to validate it locally. The build bot does validate it as well along with the desktop file & the icons.